### ISSUE TITLE
New Feature Request: Allow attaching images for comments on git Issues.

### ISSUE THREAD

**Issue by shobhit-agarwal** [slapperwan/gh4a#297]
For mobile app repos, it will be a great idea to allow to include image attachments (as provided by github.com).
Esp. for comments is issues.

**Comment by maniac103** [comment]
This doesn't seem to be supported by the Github API. The upload to Github's servers doesn't appear anywhere in the API docs, and other people came to the conclusion it's not supported as well:
http://stackoverflow.com/questions/28703544/create-an-issue-with-image-via-github-api

So not sure what to do there; we certainly don't want to do an own image hosting service.

**Comment by alorma** [comment]
You can use imgur api :P

**Comment by Bluscream** [comment]
Yeah use imgur :D

**Comment by jancborchardt** [comment]
Yep, same for creating new issues. If Stackoverflow uses imgur, it should be proper for Octodroid too. ;)

**Comment by Progi1984** [comment]
It seems that this gist has found the API https://gist.github.com/15c3fd136c0e8fd547db32ad72ab045a

**Comment by maniac103** [comment]
Nah, this gist stores all uploaded files into a certain repo. This is certainly not what we want: we want a storage alongside issues, not within the repo content.

**Comment by Progi1984** [comment]
May be a solution here :+1: 

- https://github.com/zmwangx/ghuc/blob/5f7489e821aff0a3b7dabbd5b95446a5d2459ab4/ghuc.py#L297
- https://github.com/maple3142/ghfileupl/blob/c697e56b1fb8376f0ea99543a7e704c3cdf81d6d/index.js#L87
- https://github.com/mdaliejaz/create-github-issues-with-image/blob/6fff8c9e5f5b7803710dc725f07c683c5d4d0bf1/upload_issues.rb#L25
- https://github.com/osdakira/chrome_tests/blob/99892d4c31937bfe021e9736ba28b7a8bbec1893/yammer_to_github/js/content.js#L53

**Comment by 3xploiton3** [comment]
Still waiting, this feature very important

Example: fasthub can do this issue

**Comment by 3xploiton3** [comment]
@maniac103 pls take a look again

**Comment by KaKi87** [comment]
Hello, any news on this ? Thanks

**Comment by Bluscream** [comment]
Its dead. If you need to look at code, use OctoDroid, for anything markdown related, use Fasthub

**Comment by maniac103** [comment]
It's not dead, it's just in 'maintenance onlyc mode for the foreseeable future. But wasn't development on FastHub stopped some time ago?

**Comment by KaKi87** [comment]
> Its dead.

I don't feel that the OctoDroid project is dead, I mean the proportion of my issues that have been resolved here is quite high compared to other projects.

> it's just in 'maintenance onlyc mode for the foreseeable future.

I didn't of heard that, where's the reference to this ?

Thanks

**Comment by maniac103** [comment]
> I didn't of heard that, where's the reference to this ?

The reference is a statement made by the (only) maintainer ... which is me ;-)

**Comment by KaKi87** [comment]
I know, I mean : is there a dedicated statement regarding this matter that I didn't heard of, or did you just decided it while reading and answering to this issue ?

**Comment by maniac103** [comment]
I didn't make an 'official' statement thus far, but it's not something that I decided today. It's just a description of the amount of work I'm able to put into this project for some time now. Between day job, family and other side projects I'm simply severely lacking time for anything besides maintenance :-/ I fully intend to keep doing bugfixes though, since OctoDroid is my own GH client of choice (who would have guessed) and thus I have an own interest in keeping it working.

**Comment by KaKi87** [comment]
Understood. And of course I believe you when you say your're using your own software. I mean, who, as an open source developer, would build something he wouldn't use himself ? Personally I wouldn't.

**Comment by toas-koas** [comment]
Is there any update?
